kubernetes/deployment-production.tmpl

```diff
@@ -36,4 +36,29 @@ spec:
     - protocol: TCP
       port: 80
       targetPort: 80
-  type: NodePort
\ No newline at end of file
+  type: NodePort
+
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: zoomapper-production-ingress
```

Comment:
We normally match the ingress name to the domain it's for unless it's for a set of domains (not this case)
```suggestion
  name: zoomapper.zooniverse.org
```